internal/tools/changes/.changes/releases/2020-06-26.json

```diff
@@ -0,0 +1,30 @@
+{
+ "ID": "2020-06-26",
+ "SchemaVersion": "1.0",
+ "VersionBumps": {
+  "changes": {
+   "From": "v1.0.0",
+   "To": "v1.0.1"
+  },
+  "test": {
+   "From": "v1.2.3",
+   "To": "v1.3.0"
+  }
+ },
+ "Changes": [
+  {
+   "ID": "changes-feature-1593193458080809000",
+   "SchemaVersion": "1.0",
+   "Module": "changes",
+   "Type": "feature",
+   "Description": "this is a test"
+  },
+  {
+   "ID": "test-feature-1593193458080838000",
+   "SchemaVersion": "1.0",
+   "Module": "test",
+   "Type": "feature",
+   "Description": "this is a test"
```

Comment:
Yeah, you can add markdown when creating a change. I added a test case that tests/demonstrates this. The one potential issue is that each change file is mapped to a single bullet point in the generated CHANGELOG, although in theory nothing is actually stopping a developer from entering two bullet points in the description:

```
modules:
  - module
type: feature
description: |-
  first bullet point <-- first bullet point * is already added by the tool
  * second bullet point
```

Reply:
It would be really great if we can avoid the conditional first bullet point added by tool complexity. such as if first item has bullet point accept, if not add.